fastparquet/test/test_compression.py

```diff
@@ -55,6 +55,28 @@ def test_compress_decompress_roundtrip_args_lz4():
     decompressed = decompress_data(compressed, algorithm="lz4")
     assert data == decompressed
 
+def test_compress_decompress_roundtrip_args_zstd():
+    pytest.importorskip('zstandard')
+    data = b'123' * 1000
+    compressed = compress_data(
+        data,
+        compression={
+            "type": "zstd",
+            "args": {
+                "level": 5,
+                "write_dict_id": True,
+                "threads": 0,
```

Comment:
It would be interesting to experiment with the performance now that we have so many options for the new de/compressors!